pytestqt/exceptions.py

```diff
@@ -68,12 +68,19 @@ def format_captured_exceptions(exceptions):
     Formats exceptions given as (type, value, traceback) into a string
     suitable to display as a test failure.
     """
-    message = 'Qt exceptions in virtual methods:\n'
-    message += '_' * 80 + '\n'
+    if sys.version_info.major == 2:
+        from StringIO import StringIO
+    else:
+        from io import StringIO
```

Comment:
Why not have this import at the top of the file? Might also want to do `except ImportError:` instead so you don't need `sys` if you do it at the top.

Reply:
> Why not have this import at the top of the file?

I put it here locally mostly because this is function is rarely used. If you prefer I can move it to the top though, no problem.

> Might also want to do except ImportError: instead so you don't need sys if you do it at the top.

In general I tend to avoid that because it might lead to hard to diagnose erors (see https://github.com/pytest-dev/pytest-mock/issues/68).

Reply:
@The-Compiler gentle ping. 😁

Reply:
Fair point - seems fine to keep it here.

Reply:
Thanks!